apps/services/auth-public-api/src/app/modules/delegations/national-registry.config.ts

```diff
@@ -1,17 +0,0 @@
-import { NationalRegistryModuleConfig } from '@island.is/clients/national-registry-v2'
-import { createXRoadAPIPath } from '@island.is/shared/utils/server'
-
-import { environment } from '../../../environments'
-
-export const NationalRegistryConfig: NationalRegistryModuleConfig = {
-  xRoadPath: createXRoadAPIPath(
-    environment.nationalRegistry.xroad.basePath,
-    environment.nationalRegistry.xroad.memberClass,
-    environment.nationalRegistry.xroad.memberCode,
-    environment.nationalRegistry.xroad.apiPath,
-  ),
-  xRoadClient: environment.nationalRegistry.xroad.clientId,
-  fetch: {
-    timeout: 5000,
```

Comment:
Where is this configured now?

Reply:
In config and helm. Have to add it:

![image](https://user-images.githubusercontent.com/115094/143855990-eba739fe-6c7b-40db-8355-48eceddbc086.png)